### PR TITLE
Fds 2212 test coverage

### PR DESCRIPTION
**Pull request by afwillia**

### CODE REVIEW

File: tests/testthat/test_schematic_rest_api.R [hunk 1]
```diff
@@ -20,35 +20,38 @@ test_that("manifest_generate returns a URL if sucessful", {
...
 })
 
-test_that("manifest_generate returns an xlsx", {
-  skip_it()
-  
-  xlsx <- manifest_generate(title="Test biospecimen", data_type="Biospecimen",
-                    asset_view="syn33715412", output_format="excel")
-  
-})
+# test_that("manifest_generate returns an xlsx", {
```
Comment (lakikowolfe) [inline tests/testthat/test_schematic_rest_api.R:31]: Are you intending to leave this commented out?

File: tests/testthat/test_schematic_rest_api.R [hunk 2]
```diff
@@ -58,63 +61,66 @@ test_that("model_submit successfully uploads to synapse", {
...
-  exp <- setNames(c("BulkRNA-seqAssay", "CSV/TSV", "Sample_A", "GRCm38", NA, 2022L, "syn28278954"),
-    c("Component", "File Format", "Filename", "Genome Build", "Genome FASTA", "Sample ID", "entityId"))
-  expect_equal(unlist(csv), exp)
-})
+# test_that("manifest_download returns a csv.", {
+#   skip_it()
+  csv <- manifest_download(url=file.path(schematic_url, "v1/manifest/download"),
+                           manifest_id="syn51078535",
+                           access_token=Sys.getenv("SYNAPSE_PAT"))
+#   exp <- setNames(c("BulkRNA-seqAssay", "CSV/TSV", "Sample_A", "GRCm38", NA, 2022L, "syn28278954"),
```
Comment (lakikowolfe) [inline tests/testthat/test_schematic_rest_api.R:115]: Same question about commented code as above

**Review by lakikowolfe — COMMENTED** [review]
Looks good. I'd still like to go over the gh action yml in code review just to make sure I fully understand what each step is doing. Let me know what the plan is with the commented out code and I can approve

**Comment by afwillia** [comment]
@lakikowolfe yea sorry for the complete lack of comments or context here. I am leaving the commented-out code for now because I just need these test files to run without error to finish setting up the test coverage. Then I intend to go back and update them for FDS-2214.

The gh workflow was created automatically with `usethis` and was recommended by the covr docs. Let's chat about it during CR. My goal is to have it run all the tests and create a report and/or badge.

**Review by lakikowolfe — APPROVED** [review]

**Pull request merged**